standalone-haddock.cabal

```
@@ -23,12 +23,13 @@ executable standalone-haddock
     Cabal.Simple
   -- other-extensions:    
   build-depends:
-    base == 4.*,
-    Cabal >= 1.22,
+    base >= 4.7 && < 5,
+    Cabal >= 2.2,
```

I tried to do a test build (in a cabal sandbox) and it ended up building against Cabal 2.4.0.1, which failed (missing Distribution.Simple.Utils.matchFileGlob, which does seem to have been removed in Cabal 2.4). I usually use stack myself, but I'd like to keep both options open here for the time being.
```suggestion
    Cabal >= 2.2 && < 2.4,
```